### PR TITLE
Add Content-Transfer-Encoding header

### PR DESCRIPTION
**Pull request by Cydrobolt**
Base64 emails are not being shown correctly.

### CODE REVIEW

File: fmn/consumer/backends/mail.py [hunk 1]
```diff
@@ -84,8 +86,8 @@ def send_mail(self, session, recipient, subject, content,
         # Explicitly declare encoding, but remove the transfer encoding
         # https://github.com/fedora-infra/fmn/issues/94
         email_message.set_charset('utf-8')
-        if 'Content-Transfer-Encoding' in email_message:
-            del email_message['Content-Transfer-Encoding']
+        #if 'Content-Transfer-Encoding' in email_message:
+        #    del email_message['Content-Transfer-Encoding']
```
Comment (ralphbean) [inline fmn/consumer/backends/mail.py]: It should just be sufficient to uncomment these lines as the Content-Transfer-Encoding was originally set to ``base64``.

I wasn't explicitly encoding things as base64 here, so I thought it was a mistake.  Clearly, that was wrong.

Comment (Cydrobolt) [inline fmn/consumer/backends/mail.py]: :+1:

**Comment by Cydrobolt** [comment]
I got rid of the repeated header.

**Comment by ralphbean** [comment]
:+1:

**Pull request merged**